notebooks/growth_model_forecast.ipynb

```diff
@@ -7,14 +7,23 @@
    "source": [
```

Comment:
<p>I think we should add a markdown cell before this to describe what MAPE is and why we are using it as an accuracy metric. We should also mention the mape range values and what a good/bad mape value signifies</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>